### PR TITLE
Added basic logic for .gittt.yml handling

### PR DESCRIPTION
**Pull request by eiabea**

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -66,18 +66,11 @@ export class App {
   public async setup(): Promise<void> {
     this.configDir = path.join(FileHelper.getHomeDir(), `${APP_CONFIG_DIR}`);
     this.fileHelper = new FileHelper(this.configDir, "config.json", "timer.json", "projects");
+    this.configHelper = new ConfigHelper(this.fileHelper);
 
     // TODO correct place to ask this?
```
Comment (lombad) [inline app.ts]: Remove TODO

File: helper/project.ts [hunk 2]
```diff
@@ -71,76 +75,117 @@ export class ProjectHelper {
...
+  //       // TODO should throw instead of exiting here
+  //       return process.exit(1);
+  //     }
+  //   } else {
+  //     foundProject = project;
+  //   }
+
+  //   return foundProject;
+  // }
+
```
[the remark this answers is not in the document]
Comment (eiabea) [reply helper/project.ts]: I think i will move this to a new story to implement a better migration. Also implementing a migration from `.git/config` parsing projects to `.gittt.yml` projects, which is currently only covered by a how to in the `README.md`

File: helper/project.ts [hunk 3]
```diff
@@ -283,89 +333,101 @@ export class ProjectHelper {
...
+  //         default:
+  //           LogHelper.error("✗ Invalid link type");
+  //           break;
+  //       }
+  //     }
+  //   }
+
+  //   return migratedProject;
+  // }
```
[the remark this answers is not in the document]
Comment (eiabea) [reply helper/project.ts]: As mentioned above, this will move to a new user story

File: helper/project.ts [hunk 4]
```diff
@@ -407,14 +394,12 @@ export class ProjectHelper {
 
     try {
       // const projectFromGit: IProject = this.getProjectFromGit();
```
Comment (lombad) [inline helper/project.ts]:
```suggestion

```

**Pull request merged**